modules/editor/message/MessageEditor.tsx

```diff
@@ -40,40 +56,67 @@ export function MessageEditor(props: MessageEditorProps) {
 
   return useObserver(() => (
     <Stack gap={16}>
-      <div>
-        <PrimaryContentEditor message={message} form={form} />
-        <ErrorWrapper>
+      <PrimaryContentEditor message={message} form={form} />
+      {message.embeds.length > 0 && (
+        <div>
           <InputError
             error={
               message.embedLength > 6000
                 ? "Embeds exceed 6000 character limit"
                 : undefined
             }
           />
-        </ErrorWrapper>
-      </div>
-      {message.embeds.map((embed, index) => (
-        <EmbedEditor
-          key={embed.id}
-          embed={embed}
-          form={form.repeatingForm("embeds").index(index)}
-        />
-      ))}
+          <Stack gap={16}>
+            {message.embeds.map((embed, index) => (
+              <EmbedEditor
+                key={embed.id}
+                embed={embed}
+                form={form.repeatingForm("embeds").index(index)}
+              />
+            ))}
+          </Stack>
+        </div>
+      )}
       <div>
         <PrimaryButton
           disabled={message.size >= 10}
           onClick={() => {
-            form.repeatingForm("embeds").push({} as EmbedLike, ["timestamp"])
+            form.repeatingForm("embeds").push({} as EmbedLike, ["content"])
```

Comment:
Timestamp is needed there because it's an object not known by the form system, content isn't a property on embeds
```suggestion
            form.repeatingForm("embeds").push({} as EmbedLike, ["timestamp"])
```